tools/accuracy_checker/dataset_definitions.yml

```diff
@@ -1064,3 +1064,22 @@ datasets:
       audio_dir: audio_dataset/data
     annotation: sound_classification.pickle
     reader: wav_reader
+
+  - name: cocosnet_image_translation
+    annotation_conversion:
+      converter: ade20k_image_translation
+      images_dir: ADEChallengeData2016/images
+      annotations_dir: ADEChallengeData2016/annotations
+      reference_file: cocosnet_ade20k_ref_test.txt
+    data_source: ADEChallengeData2016
+    metrics:
+      - type: psnr
+        scale_border: 0
+        presenter: print_vector
+      - type: ssim
```

Comment:
psnr and ssim do not use classification model. Is there way to use custom evaluator without it, if user wants only calculate them?

Reply:
I moved metrics to accuracy config file. Now users are able to calculate psnr and ssim metrics without calculating IS and FID metrics.